project_closing/README.rst

```diff
@@ -4,8 +4,8 @@
 Project closing
 ===============
 
-Automatic account analytic closing when related project is closed,
-and if a project is opened, the related analytic account will be re-opened.
+Automatic account analytic archiving when related project is archived,
+and if a project is dearchived, the related analytic account will be dearchived as well.
```

Comment:
I know it's annoying, but can you please recreate the README using the latest template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst